### ISSUE TITLE
Module not found: Error: Can't resolve 'react/lib/ReactCSSTransitionGroup' after version 2.4.0

### ISSUE THREAD

**Issue by Filipvdb321** [zeit/next.js#2102]
I realize this error could be specific to my application but it happens only after upgrading from Next version 2.4.0 to the latest version.

I have a dependency which has a dependency to react-addons-css-transition-group. When upgrading the latest 'next' version I get he error below when building the application. Moving back to the old version fixes the problem.

Could it be caused by the new webpack version used in the newer versions? 

**Stacktrace**` Failed to build on /var/folders/jr/dttm62vx7ss636n38lbv632h0000gn/T/362dd530-0e85-42d9-af04-7288fe481a7e
{ Error: ./~/react-addons-css-transition-group/index.js
Module not found: Error: Can't resolve 'react/lib/ReactCSSTransitionGroup' in '/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react-addons-css-transition-group'
resolve 'react/lib/ReactCSSTransitionGroup' in '/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react-addons-css-transition-group'
  Parsed request is a module
  using description file: /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react-addons-css-transition-group/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
    aliased with mapping 'react': '/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js' to '/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup'
      using description file: /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react-addons-css-transition-group/package.json (relative path: .)
        Field 'browser' doesn't contain a valid alias configuration
      after using description file: /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react-addons-css-transition-group/package.json (relative path: .)
        using description file: /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/package.json (relative path: ./dist/react.min.js/lib/ReactCSSTransitionGroup)
          as directory
            /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup doesn't exist
          no extension
            Field 'browser' doesn't contain a valid alias configuration
            /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup doesn't exist
          .js
            Field 'browser' doesn't contain a valid alias configuration
            /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup.js doesn't exist
          .json
            Field 'browser' doesn't contain a valid alias configuration
            /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup.json doesn't exist
[/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup]
[/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup]
[/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup.js]
[/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup.json]
 @ ./~/react-addons-css-transition-group/index.js 1:17-61
 @ ./~/auth0-lock/lib/ui/box/chrome.js
 @ ./~/auth0-lock/lib/ui/box/container.js
 @ ./~/auth0-lock/lib/core/actions.js
 @ ./~/auth0-lock/lib/core.js
 @ ./~/auth0-lock/lib/index.js
 @ ./utils/auth.js
 @ ./store.js
 @ ./pages/contact.js?entry
 @ multi ./pages/contact.js?entry
    at /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/next/dist/server/build/index.js:181:21
    at /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/next/node_modules/webpack/lib/Compiler.js:272:15
    at Compiler.emitRecords (/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/next/node_modules/webpack/lib/Compiler.js:367:37)
    at /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/next/node_modules/webpack/lib/Compiler.js:265:12
    at /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/next/node_modules/webpack/lib/Compiler.js:360:11
    at next (/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/tapable/lib/Tapable.js:154:11)
    at Compiler.compiler.plugin (/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/next/node_modules/webpack/lib/performance/SizeLimitsPlugin.js:99:4)
    at Compiler.applyPluginsAsyncSeries1 (/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/tapable/lib/Tapable.js:158:13)
    at Compiler.afterEmit (/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/next/node_modules/webpack/lib/Compiler.js:357:8)
    at Compiler.<anonymous> (/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/next/node_modules/webpack/lib/Compiler.js:352:14)
  errors: [ './~/react-addons-css-transition-group/index.js\nModule not found: Error: Can\'t resolve \'react/lib/ReactCSSTransitionGroup\' in \'/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react-addons-css-transition-group\'\nresolve \'react/lib/ReactCSSTransitionGroup\' in \'/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react-addons-css-transition-group\'\n  Parsed request is a module\n  using description file: /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react-addons-css-transition-group/package.json (relative path: .)\n    Field \'browser\' doesn\'t contain a valid alias configuration\n    aliased with mapping \'react\': \'/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js\' to \'/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup\'\n      using description file: /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react-addons-css-transition-group/package.json (relative path: .)\n        Field \'browser\' doesn\'t contain a valid alias configuration\n      after using description file: /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react-addons-css-transition-group/package.json (relative path: .)\n        using description file: /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/package.json (relative path: ./dist/react.min.js/lib/ReactCSSTransitionGroup)\n          as directory\n            /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup doesn\'t exist\n          no extension\n            Field \'browser\' doesn\'t contain a valid alias configuration\n            /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup doesn\'t exist\n          .js\n            Field \'browser\' doesn\'t contain a valid alias configuration\n            /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup.js doesn\'t exist\n          .json\n            Field \'browser\' doesn\'t contain a valid alias configuration\n            /Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup.json doesn\'t exist\n[/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup]\n[/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup]\n[/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup.js]\n[/Users/Filip/Documents/javascript/RestoBonsApp/RestoBonsWeb/src/node_modules/react/dist/react.min.js/lib/ReactCSSTransitionGroup.json]\n @ ./~/react-addons-css-transition-group/index.js 1:17-61\n @ ./~/auth0-lock/lib/ui/box/chrome.js\n @ ./~/auth0-lock/lib/ui/box/container.js\n @ ./~/auth0-lock/lib/core/actions.js\n @ ./~/auth0-lock/lib/core.js\n @ ./~/auth0-lock/lib/index.js\n @ ./utils/auth.js\n @ ./store.js\n @ ./pages/contact.js?entry\n @ multi ./pages/contact.js?entry' ],
  warnings: [] }
`

**Comment by timneutkens** [comment]
Duplicate of https://github.com/zeit/next.js/issues/1877
